frontend/src/component/admin/network/NetworkTrafficUsage/hooks/useTrafficLimit.ts

```diff
@@ -1,24 +0,0 @@
-import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
-import { useUiFlag } from 'hooks/useUiFlag';
-
-const proPlanIncludedRequests = 53_000_000;
-const paygPlanIncludedRequests = proPlanIncludedRequests;
-
-export const useTrafficLimit = () => {
-    const { isPro, isEnterprise, uiConfig } = useUiConfig();
-    const isEnterprisePaygEnabled = useUiFlag('enterprise-payg');
-
-    if (isPro()) {
-        return proPlanIncludedRequests;
-    }
-
-    if (
-        isEnterprisePaygEnabled &&
-        isEnterprise() &&
-        uiConfig.billing === 'pay-as-you-go'
-    ) {
```

Comment:
This one was a little odd, what would have happened if that flag was switched off for that customer and their billing was pay-as-you-go?

Reply:
You ask me, I ask you